daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-message-ttl.md

```diff
@@ -1,34 +1,34 @@
 ---
 type: docs
-title: "Message Time-To-Live"
+title: "Message Time-to-Live (TTL)"
 linkTitle: "Message TTL"
 weight: 6000
 description: "Use time-to-live in Pub/Sub messages." 
 ---
 
 ## Introduction
 
-Dapr enables per-message time-to-live (TTL). This means that applications can set time-to-live per message, and subscribers will not receive those messages after expiration.
+Dapr enables per-message time-to-live (TTL). This means that applications can set time-to-live per message, and subscribers do not receive those messages after expiration.
 
 All Dapr [pub/sub components]({{< ref supported-pubsub >}}) are compatible with message TTL, as Dapr handles the TTL logic within the runtime. Simply set the `ttlInSeconds` metadata when publishing a message.
 
 In some components, such as Kafka, time-to-live can be configured in the topic via `retention.ms` as per [documentation](https://kafka.apache.org/documentation/#topicconfigs_retention.ms). With message TTL in Dapr, applications using Kafka can now set time-to-live per message in addition to per topic.
 
-## Native message TTL support
+## Pub/sub component message TTL support
```

Comment:
I think we should keep this as "native" to convey that important part that TTL is handled natively outside of Dapr

Reply:
Reverted change to Native